platform/python/django/opentelemetry/django-postgresql.dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.9.1-buster
+
+WORKDIR /django
+
+COPY requirements.txt ./
+RUN pip install -r requirements.txt
+COPY requirements-opentelemetry.txt ./
+RUN pip install -r requirements-opentelemetry.txt
```

Comment:
Similar to what we do for Sentry instrumentation -- makes it so that all commands above this one can be reused from cache.